src/coreclr/jit/simdashwintrinsic.cpp

```diff
@@ -402,6 +402,32 @@ GenTree* Compiler::impSimdAsHWIntrinsicSpecial(NamedIntrinsic       intrinsic,
 
     switch (intrinsic)
     {
+#if defined(TARGET_XARCH)
+        case NI_VectorT128_ConvertToDouble:
+        case NI_VectorT256_ConvertToDouble:
+        case NI_VectorT128_ConvertToInt64:
+        case NI_VectorT256_ConvertToInt64:
+        case NI_VectorT128_ConvertToUInt32:
+        case NI_VectorT256_ConvertToUInt32:
+        case NI_VectorT128_ConvertToUInt64:
+        case NI_VectorT256_ConvertToUInt64:
+        {
+            // TODO-XARCH-CQ: These intrinsics should be accelerated
```

Comment:
There isn't an issue for the intrinsics specifically, but a broader issue under which this would get touched.

For reference, this is something that already wasn't accelerated for `Vector<T>` and where we could accelerate it, but doing so requires some more complex logic and potentially normalizing the behavior of floating-point conversions to be deterministic across all hardware: https://github.com/dotnet/runtime/issues/61885

Reply:
> For reference, this is something that already wasn't accelerated for ...

That was my understanding after the code review, thanks for confirming.